### PR TITLE
Rv asmft 353

### PR DESCRIPTION
**Pull request by robertvazquezlegalshield**
This is to view the spouse/partner listed for a member in the Partner app.  Also included is a mods file for Adonis which includes a function to simplify the login to the Partner app.

### CODE REVIEW

File: demo/adonis/partner/view_spouse.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+var i = require('../../tools.js')
+var secure = require('../../secret.js')
+var p = require('../../adonis_mods.js')
+
+who('As a Partner I can', function() {
+where('https://partner.sandbox.legalshield.com')
+what('View the correct search results ASMFT-350', function () {
+  p.partnerLogin()
```
Comment (TrentonWalkup) [inline demo/adonis/partner/view_spouse.js]: good!

File: demo/adonis/partner/view_spouse.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+var i = require('../../tools.js')
+var secure = require('../../secret.js')
```
Comment (TrentonWalkup) [inline demo/adonis/partner/view_spouse.js]: do we need this in here now?

Comment (Gregorma) [reply demo/adonis/partner/view_spouse.js]: Oops, no he does not

**Comment by Gregorma** [comment]
LGTM!

File: demo/adonis/partner/view_spouse.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+var p = require('../../adonis_mods.js')
+
+who('As a Partner I can', function() {
```
Comment (Bradern8t0r) [inline demo/adonis/partner/view_spouse.js]: 'As a partner I can' -What?

Comment (TrentonWalkup) [reply demo/adonis/partner/view_spouse.js]: so his 'who' leads directly into his 'what'. As a Partner I can view the correct search results ASMFT-350

File: demo/adonis/partner/view_spouse.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+where('https://partner.sandbox.legalshield.com')
+what('View the correct search results ASMFT-350', function () {
```
Comment (TrentonWalkup) [inline demo/adonis/partner/view_spouse.js]: Your branch is 353 but this says 350?

File: demo/adonis/partner/view_spouse.js (hunk 3 above)
Comment (Bradern8t0r) [reply demo/adonis/partner/view_spouse.js]: Gotcha.

**Comment by robertvazquezlegalshield** [comment]
Corrected the "what" and also removed var secure = require('../../secret.js')

**Comment by robertvazquezlegalshield** [comment]
Ok'd by Trenton and Mercedes

**Pull request merged**